src/profiler.cpp

```diff
@@ -83,6 +86,29 @@ void Profiler::add(const std::string &name, float value)
 	}
 }
 
+void Profiler::max(const std::string &name, float value)
+{
+	MutexAutoLock lock(m_mutex);
+	{
+		/* No average shall have been used; mark add used as -2 */
+		std::map<std::string, int>::iterator n = m_avgcounts.find(name);
```

Comment:
```suggestion
		auto n = m_avgcounts.find(name);
```

Reply:
+1

Just used the same code from add.

Reply:
In fact that whole -2 business is not needed and just slows things down. But I'll leave it in place.